docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,150 @@
+
+## Setting up a workflow
+
+Create a `.circleci/config.yml` file in the root of your repository and follow these steps:
+
+1. Import the [Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity). It's recommended
+   to fix a `minor` or `patch` version to avoid updates with breaking changes:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+```
+
+2. Include a [build job](/docs/circleci/build) in your workflow and modify the
+   [parameters](/docs/circleci/build#parameters) to accommodate your project needs:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          name: 'build-linux64-il2cpp'
+          step-name: 'Build StandaloneLinux64'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: StandaloneLinux64
+          compress: true
+          context: unity
+```
+
+3. Optionally, include a [test job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-test)
+   in your workflow. You can add it as a
+   [requirement](https://circleci.com/docs/configuration-reference#requires) in the build job to
+   prevent it from running in case of failure:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  test-and-build-unity-project:
+    jobs:
+      - unity/test:
+          name: 'test-linux'
+          step-name: 'Check if tests run and results are uploaded'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'medium'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          test-platform: 'playmode'
+          context: unity
+      - unity/build:
+          name: 'build-linux64-il2cpp'
+          step-name: 'Build StandaloneLinux64'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: StandaloneLinux64
+          compress: true
+          context: unity
+          requires:
+            - test-linux
+```
+
+4. Commit and push the new workflow.
+
+## Downloading the artifacts
```

Comment:
```suggestion
## Downloading the result
```

Reply:
I feel like artifacts make more sense because that is how CircleCI calls the data you store throughout the job.